docs/extensions.rst

```diff
@@ -7,7 +7,16 @@ It's easy to extend the xlwings add-in with own code like UDFs or RunPython macr
 without end users having to import or write the functions themselves. Just add another VBA module to the xlwings addin
 with the respective code.
 
-UDF extensions can be used from every workbook without having to set a reference. 
+UDF extensions can be used from every workbook without having to set a reference.
+
+Using RunPython to run an extension function requires you to import xlwings.ext.
+
+.. code::
+
+  Sub LoadKoalaModel()
+    RunPython ("import xlwings.ext as xwe; xwe.Myfunction()")
+  End Sub
+
```

Comment:
That doesn't make sense to me? Extensions refer to a storing the VBA wrappers within the addin. The python source code (except for functions that we maintain as part of the official repo, i.e. for now sql) should live anywhere on the pythonpath, but certainly not in xlwings.ext.

Reply:
I had attempted to write an extension and found that was the only way I could make it work. The extension I write ended up being not required. I may try again in the future and will be happy to accept advice on how to write an extension well enough to be considered.

In aid of getting the rest of the pull request accepted I'm happy to omit the change to docs/extensions.rst.

Reply:
The python part of an extension needs to be found by xlwings either because it's on the PYTHONPATH anyways (e.g. by placing it in the `site-packages` dir of your installation) or by setting the PYTHONPATH in the xlwings addin/config to a dir where the source code is.
Don't worry about it, I'll try tackle this this week so will take care of this.

Reply:
Thank you for taking the time to steer me in the right direction. I have had a few minutes to sit quietly and digest your comments. As always, your advice is exceptional.

I have re-visited my attempt at extension and I now know where I've gone wrong with suggesting that particular change to the docs.